### DISCUSSION TITLE
[Bug] Compiling XF from source

### DISCUSSION THREAD

**Discussion by AlleSchonWeg** [xamarin/Xamarin.Forms#11464] [General]
### Description
Hi,
i pulled the current main branch and try to compile it. But i got the following error:
![Error](https://user-images.githubusercontent.com/6727366/87766000-d239e000-c818-11ea-8789-2d04dd89e116.PNG)

This a autogenerated file. But if i change the file (comment out) vs compiles the solution and i can deploy to android emulator.
![Unbenannt](https://user-images.githubusercontent.com/6727366/87766273-2cd33c00-c819-11ea-8d58-aefc58fefa5c.PNG)

### Basic Information

- IDE: VS 2019 16.6.4

**Comment by hartez** [comment]
Possibly related: https://developercommunity.visualstudio.com/content/problem/1055833/duplicate-globalsystemruntimeversioningtargetframe.html
